tests/py/tests/test_ethereum_jsonrpc.py

```diff
@@ -0,0 +1,73 @@
+from common.chain_tester.chain_tester import ChainTester
+from common.chain_tester.misc import TransactionExpectation
+from common.eth.solidity import compile_single
+from common.util.pytest import treat_as_warning
+
+
+@treat_as_warning(TimeoutError)
```

Comment:
@Taraxa-project/ledger Made this test failure always non-fatal if a TimeoutError ocurred